src/figuresClasses.ts

```diff
@@ -1,11 +1,88 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
 
-export class Triangle implements Figure {}
+  color: string;
 
-export class Circle implements Figure {}
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: string;
+
+  color: string;
+
+  constructor(
+    color: string,
+    public a: number = 0,
+    public b: number = 0,
+    public c: number = 0,
+  ) {
+    if (a + b <= c || c + b <= a || a + c <= b) {
+      throw new Error('its not triangle');
+    }
+    this.color = color;
+    this.shape = 'triangle';
+    this.a = a;
+    this.b = b;
+    this.c = c;
+  }
+
+  getArea(): number {
+    const sum = (this.a + this.b + this.c) / 2;
+
+    return +Math.sqrt(
+      sum * (sum - this.a) * (sum - this.b) * (sum - this.c),
+    ).toFixed(2);
+  }
+}
+
+export class Circle implements Figure {
+  shape: string;
+
+  color: string;
+
+  radius: number;
+
+  constructor(color: string, radius: number) {
+    if (radius <= 0) {
+      throw new Error('its not Circle');
```

Comment:
The error message 'its not Circle' is not descriptive enough. Consider providing more detail, such as 'Circle radius must be greater than zero.'